qa/automation/CareerPortal.Tests.Scenarios/3. Send CV/3.1. Send CV.feature

```diff
@@ -0,0 +1,126 @@
+﻿Feature: 3.1. Send CV
+
+Background: 
+	Given User is on the Job description page of the Carrier Portal
+	And the Send CV form is visible
```

Comment:
Maybe you should also add that CV form is clear

Reply:
Yes, I will add it